### PR TITLE
[bitnami/etcd] Add persistent volume claim selector

### PR DESCRIPTION
**Pull request by leszekbulawa**
**Description of the change**

Add option to specify a label selector in persistent volume claim.
ref: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#selector

**Benefits**

In some cases there is a pool of pre-existing persistent volumes and a need of use only specific type. This can be achieved by adding labels to the PVs and selector to PVCs.

**Possible drawbacks**

**Applicable issues**

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/etcd/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: etcd
-version: 4.9.5
+version: 4.9.6
```
Comment (juan131) [inline bitnami/etcd/Chart.yaml:3]: Please bump minor version instead since we're adding a new parameter.

Comment (leszekbulawa) [reply bitnami/etcd/Chart.yaml]: fixed

File: bitnami/etcd/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: etcd
-version: 4.9.5
+version: 4.10.5
```
Comment (juan131) [inline bitnami/etcd/Chart.yaml:3]: This should be `4.10.0`

Comment (leszekbulawa) [reply bitnami/etcd/Chart.yaml:3]: sorry, looks like I need a coffe :)

Comment (leszekbulawa) [reply bitnami/etcd/Chart.yaml]: amended

Comment (juan131) [reply bitnami/etcd/Chart.yaml]: Don't worry 😄

**Pull request merged**